src/common/k8s-api/kube-object.ts

```diff
@@ -197,25 +206,8 @@ export class KubeObject<Metadata extends KubeObjectMetadata = KubeObjectMetadata
     return Object.entries(labels).map(([name, value]) => `${name}=${value}`);
   }
 
-  /**
-   * These must be RFC6902 compliant paths
-   */
-  private static readonly nonEditablePathPrefixes = [
-    "/metadata/managedFields",
-    "/status",
-  ];
-  private static readonly nonEditablePaths = new Set([
-    "/apiVersion",
-    "/kind",
-    "/metadata/name",
-    "/metadata/selfLink",
-    "/metadata/resourceVersion",
-    "/metadata/uid",
-    ...KubeObject.nonEditablePathPrefixes,
-  ]);
```

Comment:
With patch, this isn't necessary anymore because the kube server will enforce this.